### PR TITLE
cl-warp: Synchronize warp config parameters with processed image buffer

### PR DESCRIPTION
**Pull request by zongwave**
Save warp config parameters into list to synchronize with image processing pipeline

### CODE REVIEW

**Comment by zongwave** [comment]
the warp config parameter comes from other thread, need to do synchronize with image processing pipeline.

File: xcore/base/xcam_smart_result.h [hunk 1]
```diff
@@ -63,7 +63,7 @@ typedef struct _XCamDVSResult {
     int                   frame_id;
     int                   frame_width;
     int                   frame_height;
-    int                   valid;
+    bool                  valid;
```
Comment (windyuan) [inline xcore/base/xcam_smart_result.h]: we still need keep 

     int valid;
since all base/xcam*.h files are C mode

**Comment by zongwave** [comment]
invoke CLImageHandler::is_ready function to synchronize warp config with processed image

**Comment by windyuan** [comment]
merged. thanks.
commit 8d3d7b7b7e0ecbcfbc628bad330751845eddf728
commit 78edd76fc8b8171765ff28df44511c151e77b1cd

**Pull request closed without merging**